drgadget/ropviewer.py

```diff
@@ -360,17 +353,15 @@ def OnKeydown(self, vkey, shift):
             elif vkey == ord("S"):
                 self.export_binary()
 
-
-        # colon
-        elif vkey == 190:
+        elif vkey == 186:  # colon
```

Comment:
Also, I have a suggestion: it seems that the reason the colon is problematic is that it doesn't have a fixed location in different keyboard layouts (I've seen in referenced as an OEM-specific key in a couple of places). I think we won't have this issue with the Insert key, which in IDA is also used for comments. I receive the Insert key as vkey=45; if you agree with my suggestion could you check that it works the same for you?